tensorflow_datasets/dataset_builder_testing_test.py

```diff
@@ -0,0 +1,35 @@
+# coding=utf-8
+# Copyright 2020 The TensorFlow Datasets Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for tensorflow_datasets.testing.dataset_builder_testing."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import tensorflow.compat.v2 as tf
+import tensorflow_datasets.public_api as tfds
+
+
+class DatasetBuilderTesting(tf.test.TestCase):
+
+  def test_dataset_builder_testing(self):
+
+    # pylint: disable=unreachable
+    self.assertIsNotNone(tfds.testing)
+
+
+if __name__ == '__main__':
+  tf.test.main()
```

Comment:
Rather than using the `tf.test.main`, you should use our custom main, similarly to our other tests